package.json

```diff
@@ -11,14 +11,15 @@
     "build:es": "rm -rf es && cross-env BABEL_ENV=es babel src -d es --extensions .ts,.tsx --ignore src/**/*.test.tsx",
     "build:ts": "tsc --rootDir src -d --declarationDir lib && tsc --rootDir src -d --declarationDir es",
     "build:icon": "rm -rf src/General/Icon/components && babel-node --extensions .ts,.tsx src/Utils/Icon/generateIcons.ts && npm run format:icon",
-    "format": "prettier --write src/**/*.{ts,tsx} stories/**/*.{ts,tsx}",
     "format:icon": "prettier --write src/General/Icon/components/*.tsx",
-    "prepublishOnly": "npm run build",
+    "prepublishOnly": "npm run lint && npm run test && npm run build && npm whoami && npm version patch -m 'Bump to %s'",
```

Comment:
can we make bumping version an interactive command like this, let it default to bump patch number, but we can still type an arbitrary version?
```
// after lint, test and build
current version: x.x.x
bump version to (x.x.x+1):
```

Reply:
I was thinking use `npm version <patch | minor | major>` to bump version and automatically trigger publishing, and we still can just use `npm publish` directly to publish without  bumping version.

Reply:
ok, just provide a way to let us bump version manually, and add instructions in wiki.